backend/app/main/schemas.py

```diff
@@ -98,3 +106,19 @@ class User(UserBase):
 
     class Config:
         orm_mode = True
+
+
+class TokenBase(BaseModel):
+    access_token: str
+    token_type: str
+
+
+class TokenCreate(TokenBase):
+    pass
```

Comment:
Awesome job @atykwonderland! Do we need this class btw?

Reply:
oops forgot to take tht out